ServerCore/Pages/Teams/MergeInto.cshtml.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using ServerCore.DataModel;
+using ServerCore.Helpers;
+using ServerCore.ModelBases;
+
+namespace ServerCore.Pages.Teams
+{
+    [Authorize("IsEventAdmin")]
+    public class MergeIntoModel : EventSpecificPageModel
+    {
+        public MergeIntoModel(PuzzleServerContext serverContext, UserManager<IdentityUser> userManager) : base(serverContext, userManager)
+        {
+        }
+
+        [BindProperty]
+        public Team Team { get; set; }
+
+        [BindProperty]
+        public int MergeIntoID { get; set; }
+
+        public List<Team> OtherTeams { get; set; }
+
+        public async Task<IActionResult> OnGetAsync(int teamId)
+        {
+            Team = await _context.Teams.FirstOrDefaultAsync(m => m.ID == teamId);
+
+            if (Team == null)
+            {
+                return NotFound();
+            }
+
+            OtherTeams = await _context.Teams.Where(m => m.EventID == Team.EventID && m.ID != teamId).OrderBy(m => m.Name).ToListAsync();
+
+            return Page();
+        }
+
+        public async Task<IActionResult> OnPostAsync(int teamId)
+        {
+            Team = await _context.Teams.FindAsync(teamId);
```

Comment:
Do we want validation to ensure that merging teams won't exceed the member limit?

Reply:
Merging teams makes fewer teams, so no.

Reply:
I meant the individual team member limit. IE if a 6 person team merges with a 7 person team.